velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -157,16 +161,7 @@ bool ExpressionVerifier::verify(
     exec::EvalCtx evalCtxSimplified(
         execCtx_, &exprSetSimplified, rowVector.get());
 
-    try {
-      exprSetSimplified.eval(rows, evalCtxSimplified, simplifiedEvalResult);
-    } catch (...) {
-      if (!canThrow) {
-        LOG(ERROR)
-            << "Simplified eval wasn't supposed to throw, but it did. Aborting.";
-        throw;
-      }
-      exceptionSimplifiedPtr = std::current_exception();
-    }
+    exprSetSimplified.eval(rows, evalCtxSimplified, simplifiedEvalResult);
```

Comment:
Why we skip checking canThrow for exprSetSimplified and throw? Thanks!

Reply:
We come here with canThrow == false only if common path didn't throw. If simplified path throws we'll fail the Fuzzer because common path didn't throw, but simplified did. Hence, there is no need to check canThrow for simplified path.